packages/vite/src/node/watch.ts

```diff
@@ -10,9 +10,24 @@ export function resolveChokidarOptions(
 
   const resolvedWatchOptions: WatchOptions = {
     ignored: [
-      '**/.git/**',
-      '**/node_modules/**',
-      '**/test-results/**', // Playwright
+      ...[
+        '.git/**',
+        '.git*',
+        '.github/**',
+        '.vscode/**',
+        '.stackblitz/**',
+        'node_modules/**',
+        'test-results/**', // Playwright
+        'coverage/**',
+        '.editorconfig',
+        '.eslint*',
+        '.prettier*',
+        '.npmrc',
+        '.yarnrc',
```

Comment:
```suggestion
        '.yarnrc',
        '.ignore',
```

config for ripgrep, ack, ag, ...